services/web/server/src/simcore_service_webserver/projects/_projects_repository.py

```diff
@@ -115,6 +115,23 @@ async def get_project(
         return ProjectDBGet.model_validate(row)
 
 
+async def get_project_with_workbench(
+    app: web.Application,
+    connection: AsyncConnection | None = None,
+    *,
+    project_uuid: ProjectID,
+) -> ProjectWithWorkbenchDBGet:
+    async with pass_or_acquire_connection(get_asyncpg_engine(app), connection) as conn:
+        query = sql.select(*PROJECT_DB_COLS, projects.c.workbench).where(
```

Comment:
just wondering the project workbench should not be read from the other table? sorry I always forget the state of this